src/core/client/admin/test/moderate/regularQueue.spec.tsx

```diff
@@ -804,3 +812,69 @@ it("rejects comment in reported queue", async () => {
   );
   expect(within(reportedCount).getByText("1")).toBeVisible();
 });
+
+it.only("doesnt show comments from banned users whose commens have been rejected", async () => {
+  const { context } = await createTestRenderer({
+    resolvers: createResolversStub<GQLResolver>({
+      Query: {
+        moderationQueues: () =>
+          pureMerge(emptyModerationQueues, {
+            reported: {
+              count: 2,
+              comments:
+                createQueryResolverStub<ModerationQueueToCommentsResolver>(
+                  ({ variables }) => {
+                    expectAndFail(variables).toEqual({
+                      first: 5,
+                      orderBy: "CREATED_AT_DESC",
+                    });
+                    return {
+                      edges: [
+                        {
+                          node: reportedComments[0],
+                          cursor: reportedComments[0].createdAt,
+                        },
+                        {
+                          node: reportedComments[1],
+                          cursor: reportedComments[1].createdAt,
+                        },
+                      ],
+                      pageInfo: {
+                        endCursor: reportedComments[1].createdAt,
+                        hasNextPage: false,
+                      },
+                    };
+                  }
+                ) as any,
+            },
+          }),
+      },
+      Mutation: {
+        banUser: createMutationResolverStub<MutationToBanUserResolver>(
+          ({ variables }) => {
+            return {};
+          }
+        ),
+      },
+    }),
+  });
+
+  const modCard = await screen.findByTestId(
+    `moderate-comment-card-${reportedComments[0].id}`
+  );
+
+  expect(modCard).toBeInTheDocument();
+
+  act(() => {
```

Comment:
I timeboxed my attempt to open the ban modal via keyboard shortcut, but if anyone has done this successfully, let me know!